extensions/mssql/src/objectManagement/ui/objectPropertiesDialog.ts

```diff
@@ -0,0 +1,91 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+import * as azdata from 'azdata';
+import { ObjectManagementDialogBase, ObjectManagementDialogOptions } from './objectManagementDialogBase';
+import { IObjectManagementService, ObjectManagement } from 'mssql';
+import * as localizedConstants from '../localizedConstants';
+import { AlterUserDocUrl, CreateUserDocUrl } from '../constants';
+
+export class ServerPropertiesDialog extends ObjectManagementDialogBase<ObjectManagement.ServerPropertiesInfo, ObjectManagement.ServerPropertiesViewInfo> {
+	private generalTab: azdata.window.DialogTab;
+	private generalSection: azdata.GroupContainer;
+	private nameInput: azdata.InputBoxComponent;
+	private languageDropdown: azdata.DropDownComponent;
+	private memoryInput: azdata.InputBoxComponent;
+	private operatingSystemInput: azdata.InputBoxComponent;
+
+	private memoryTab: azdata.window.DialogTab;
+	private memorySection: azdata.GroupContainer;
+	private minServerMemoryInput: azdata.InputBoxComponent;
+	private maxServerMemoryInput: azdata.InputBoxComponent;
+
+	private numberInputType: azdata.InputBoxInputType = 'number';
+
+	constructor(objectManagementService: IObjectManagementService, options: ObjectManagementDialogOptions) {
+		super(objectManagementService, options);
+		this.generalTab = azdata.window.createTab(localizedConstants.GeneralSectionHeader);
+		this.memoryTab = azdata.window.createTab(localizedConstants.MemoryText);
+	}
+
+	protected override get helpUrl(): string {
+		return this.options.isNewObject ? CreateUserDocUrl : AlterUserDocUrl;
+	}
+
+	protected override async validateInput(): Promise<string[]> {
```

Comment:
You don't need to do this override, it'll already call the base implementation without it

Reply:
I get a syntax error when I remove it.

Reply:
What was the error?

Reply:
This member must have an 'override' modifier because it overrides a member in the base class

Reply:
Oh, yeah so I was saying you should delete this whole function. It's unnecessary to redefine here.

Reply:
You would only do this if you wanted to add/change the functionality of the base implementation.

Reply:
> We have the noImplicitOverride option set to true in the tsconfig file

Yeah, this is very much on purpose and a good thing to have 😄 That shouldn't apply in this case though

Reply:
Yup, I misunderstood and thought I had to remove the overwrite keyword. Just updated my pr 😃!